modules/trusted-firmware-m/Kconfig.peripheral_secure

```diff
@@ -357,12 +357,50 @@ peripheral_depends=$(dt_nodelabel_has_compat,gpio2,$(DT_COMPAT_NORDIC_NRF_GPIO))
 rsource "Kconfig.template.peripheral_secure"
 
 config NRF_GPIOTE0_SECURE
-	bool "GPIOTE configured as secure peripheral"
+	bool "GPIOTE0 configured as secure peripheral"
 	depends on $(dt_has_compat,$(DT_COMPAT_NORDIC_NRF_GPIOTE))
 	help
 	  GPIOTE0 is implemented with secure only security.
 	  This option only enables it to be used in the secure image.
 
+config NRF_GPIOTE20_SECURE
+	bool "GPIOTE20 configured as secure peripheral"
+	depends on $(dt_nodelabel_has_compat,gpiote20,$(DT_COMPAT_NORDIC_NRF_GPIOTE))
+	help
+	  GPIOTE20 is implemented with split securety. If it has been configured as
+	  non-secure it can be accessed by both secure and non-secure accesses.
+	  A bitmask can be set to configure a subset of the channels as
+	  secure to prevent non-secure from accessing these channels. Secure
+	  accesses can be made to the GPIOTE channels configured as non-secure.
+
+config NRF_GPIOTE20_SECURE_CHANNELS_MASK
+	hex "Set GPIOTE20 mask for Secure channels"
+	default 0x00000000
+	depends on $(dt_nodelabel_has_compat,gpiote20,$(DT_COMPAT_NORDIC_NRF_GPIOTE))
+	help
+	  GPIOTE supports settings individual channels as secure and non-secure. This mask is used
+	  to set the secure channels in the GPIOTE20. The mask is also used to set the interrupt
+	  security for secure channel.
+
+config NRF_GPIOTE30_SECURE
+	bool "GPIOTE30 configured as secure peripheral"
+	depends on $(dt_nodelabel_has_compat,gpiote30,$(DT_COMPAT_NORDIC_NRF_GPIOTE))
+	help
+	  GPIOTE30 is implemented with split securety. If it has been configured as
+	  non-secure it can be accessed by both secure and non-secure accesses.
+	  A bitmask can be set to configure a subset of the channels as
+	  secure to prevent non-secure from accessing these channels. Secure
+	  accesses can be made to the GPIOTE channels configured as non-secure.
+
+config NRF_GPIOTE30_SECURE_CHANNELS_MASK
+	hex "Set GPIOTE30 mask for Secure channels"
+	default 0x00000000
+	depends on $(dt_nodelabel_has_compat,gpiote30,$(DT_COMPAT_NORDIC_NRF_GPIOTE))
+	help
+	  GPIOTE supports settings individual channels as secure and non-secure. This mask is used
+	  to set the secure channels in the GPIOTE30. The mask is also used to set the interrupt
+	  security for secure channel.
+
 config NRF_GPIO0_PIN_MASK_SECURE
 	hex "Set GPIO0 PERM mask for Secure"
 	default 0x00000003 if TFM_SECURE_UART30 # P0.0 and P0.1 are used by default when TFM_SECURE_UART30
```

Comment:
Apply the following edit to NRF_VMC_SECURE help text:
```
	  When using security by separation, the non-secure part of the application should not be able to turn off the RAM block of Trusted Firmware-M.
	  Therefore, the VMC peripheral needs to be configured as secure.
```

Reply:
I will remove the "in the SPE" part because it can be confusing I think. But I will update the rest.